var/spack/repos/builtin/packages/py-jupyter-telemetry/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyJupyterTelemetry(PythonPackage):
+    """Jupyter Telemetry enables Jupyter Applications to record events and transmit"""
+    """ them to destinations as structured data"""
+
+    pypi = "jupyter-telemetry/jupyter_telemetry-0.1.0.tar.gz"
+
+    version('0.1.0', sha256='445c613ae3df70d255fe3de202f936bba8b77b4055c43207edf22468ac875314')
+    version('0.0.5', sha256='d3eaac14be17510a4d288f3737580107ce14eef543e6133d56654d3f0e742b9b')
+
+    depends_on('python@3.5:', type=('build', 'run'))
+    depends_on('py-setuptools', type=('build', 'run'))
```

Comment:
I don't know. I was copying the `depends_on(py-setuptools, ...)` from `py-jupyterhub` :|

I don't even know precisely what py-setuptools does... other than "setup" :|

Reply:
This has some good docs: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#setuptools

It's definitely needed at build-time because it's in the `setup.py`. At run-time, you'll have to see if `setuptools` or `pkg_resources` gets imported in any of the installed files. Usually it's only used in `prefix.bin` scripts but I don't see any entrypoints in this package.

Reply:
Perfect. Changed to build time only. Thank you.

Reply:
The only thing left on this is `python-json-logger` and it isn't in Spack yet. It's a weird package. No requirements.txt, empty install_requires.

Reply:
That just means that it doesn't have any dependencies.

Reply:
I think this is now resolved.